### ISSUE TITLE
Encoding error: Cannot invoke "java.util.LinkedHashMap.get(Object)" because the return value of "org.locationtech.jts.geom.Geometry.getUserData()"

### ISSUE THREAD

**Issue by springmeyer** [maplibre/maplibre-tile-spec#73]
When running the CLI locally like:

> › java -jar ./converter/java/build/libs/mvt2mlt-1.0-SNAPSHOT.jar -mvt test/fixtures/amazon/5_16_11.pbf -mlt test/expected/amazon/5_16_11.mlt

It hits an error like:

```
java.lang.NullPointerException: Cannot invoke "java.util.LinkedHashMap.get(Object)" because the return value of "org.locationtech.jts.geom.Geometry.getUserData()" is null
	at com.mlt.converter.mvt.MvtUtils.lambda$getIds$2(MvtUtils.java:109)
	at java.base/java.util.stream.ReferencePipeline$5$1.accept(ReferencePipeline.java:231)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
	at com.mlt.converter.mvt.MvtUtils.getIds(MvtUtils.java:110)
	at com.mlt.converter.mvt.MvtUtils.decodeMvt(MvtUtils.java:48)
	at com.mlt.converter.mvt.MvtUtils.decodeMvt(MvtUtils.java:24)
	at com.mlt.converter.MltCliAdapter.main(MltCliAdapter.java:189)
```

**Comment by springmeyer** [comment]
seems related to #58 / #57

**Comment by springmeyer** [comment]
Solved by #111
